src/toast-container.tsx

```diff
@@ -8,10 +8,11 @@ import {
     SafeAreaView,
     Pressable,
     Text,
+    ScrollView,
 } from 'react-native';
 import Toast, { ToastOptions, ToastProps } from './toast';
 
-const { width } = Dimensions.get('window');
+const { height, width } = Dimensions.get('window');
```

Comment:
this is a one time call, what if the window size changes? (web screen gets resized=